aws/resource_aws_rds_cluster.go

```diff
@@ -222,6 +224,29 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"iam_role": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"role_arn": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"role_status": {
+							Type:     schema.TypeString,
+							Computed: true,
```

Comment:
@stack72 did you mean this PR? https://github.com/hashicorp/terraform/pull/10822 I can't find any suggestion around status.

I can't think of an example where this field would be useful - and if there's one, then Set will effectively prevent users from doing so due to indexes (i.e. I agree with @apparentlymart ).